src/sources/file/line_agg.rs

```diff
@@ -248,6 +248,12 @@ where
     }
 }
 
+fn make_mut(bytes: Bytes) -> BytesMut {
+    let mut bytes_mut = BytesMut::new();
+    bytes_mut.extend_from_slice(&bytes);
+    bytes_mut
```

Comment:
For the record, this is fine, the copying was there before: https://docs.rs/bytes/0.4.12/src/bytes/bytes.rs.html#1653-1658.

Reply:
Yes. it was. But there also was possibility that clone will not actually happen..